### PR TITLE
Jose

### PR DESCRIPTION
**Pull request by AngelsG**
Add Jose to website

### CODE REVIEW

File: src/_data/team.yml [hunk 1]
```diff
@@ -926,4 +926,15 @@
...
+
+- id: jose_rh
+  name: José E Rodriguez
+  image: /assets/custom/img/team/jose_e_rodriguez.jpg
+  role: Managing Director (Spain)
+  fullDescription:
+    <p>I lead and support a team of exceptional people whose purpose is to accelerate organizations and to help them reach their goals. Together we help organizations tackle the challenges they face, guiding and mentoring them as they evolve by improving how they work.</p>
+
+  twitter: jrhuerta
+  github:
```
Comment (damianpumar) [inline src/_data/team.yml]: We should remove the github tag if it is empty

File: src/_data/team.yml [hunk 2]
```diff
@@ -926,4 +926,15 @@
...
-  office: london
\ No newline at end of file
+  office: london
+
+- id: jose_rh
+  name: José E Rodriguez
+  image: /assets/custom/img/team/jose_e_rodriguez.jpg
+  role: Managing Director (Spain)
+  fullDescription:
+    <p>I lead and support a team of exceptional people whose purpose is to accelerate organizations and to help them reach their goals. Together we help organizations tackle the challenges they face, guiding and mentoring them as they evolve by improving how they work.</p>
```
Comment (keith-smale) [inline src/_data/team.yml]: I would separate the sentances into separate paragraph tags so it will be split up a little more.

**Review by keith-smale — CHANGES_REQUESTED** [review]
Couple of changes to the text.

Also can the image size be adjusted to 400px by 400px.

File: src/_data/team.yml (hunk 1 above)
Comment (AngelsG) [reply src/_data/team.yml]: Solved, thanks

**Comment by keith-smale** [comment]
Looks all good. There are some conflicts from a PR that was merged just now. Can you resolve those then we can get this added....finally.

**Review by keith-smale — APPROVED** [review]

File: src/_data/team.yml (hunk 2 above)
Comment (AngelsG) [reply src/_data/team.yml]: Done, thanks!

**Pull request merged**